legend-engine-external-shared-format-model/src/main/java/org/finos/legend/engine/external/shared/format/model/api/ExternalFormats.java

```diff
@@ -60,8 +62,8 @@
 {
     private static final ObjectMapper objectMapper = ObjectMapperFactory.getNewStandardObjectMapperWithPureProtocolExtensionSupports();
     private static final Logger LOGGER = org.slf4j.LoggerFactory.getLogger("Legend Execution Server");
-    static final Map<String, ExternalFormatExtension> extensions = ExternalFormatExtensionLoader.extensions();
 
+    public static final Map<String, ExternalFormatExtension<?>> extensions = ExternalFormatExtensionLoader.extensions();
     private final ModelManager modelManager;
 
     public ExternalFormats(ModelManager modelManager)
```

Comment:
backward Incompatible changes to API. I don't think these are used yet, we will need corresponding changes on studio